src/core/adapter/devhub-contract.jsx

```diff
@@ -113,30 +113,30 @@ function getAvailableAddons() {
       configurator_widget:
         "${REPL_DEVHUB}/widget/devhub.entity.addon.telegram.Configurator",
     },
-    {
-      id: "github",
-      title: "Github",
-      description: "Connect your github",
-      view_widget: "${REPL_DEVHUB}/widget/devhub.entity.addon.github.Viewer",
-      configurator_widget:
-        "${REPL_DEVHUB}/widget/devhub.entity.addon.github.Configurator",
-    },
-    {
-      id: "kanban",
-      title: "Kanban",
-      description: "Connect your github kanban board",
-      view_widget: "${REPL_DEVHUB}/widget/devhub.entity.addon.kanban.Viewer",
-      configurator_widget:
-        "${REPL_DEVHUB}/widget/devhub.entity.addon.kanban.Configurator",
-    },
-    {
-      id: "blog",
-      title: "Blog",
-      description: "Create a blog for your community",
-      view_widget: "${REPL_DEVHUB}/widget/devhub.entity.addon.blog.Viewer",
-      configurator_widget:
-        "${REPL_DEVHUB}/widget/devhub.entity.addon.blog.Configurator",
-    },
+    // {
```

Comment:
would we like to have commented code like this merged?

Reply:
It get's uncommented when the plugins get merged; I can clean it out if we want, but it get's added right back in until #358 gets completed, but this is no blocker